packages/fxa-admin-server/src/gql/account/account.resolver.spec.ts

```diff
@@ -182,14 +217,27 @@ describe('AccountResolver', () => {
     expect(result[0].uid).toEqual(RECOVERY_KEY_1.uid);
   });
 
-  it('loads sessionTokens', async () => {
+  it('loads attached clients', async () => {
     const user = (await resolver.accountByEmail(
       USER_1.email,
       'joe'
     )) as Account;
-    const result = await resolver.sessionTokens(user);
+    const result = await resolver.attachedClients(user);
     expect(result).toBeDefined();
-    expect(result.length).toBe(1);
-    expect(result[0].tokenId).toEqual(SESSION_TOKEN_1.tokenId);
+    expect(result.length).toBe(2);
+
+    // Session, with known device, device and session merge.
+    expect(result[0].name).toEqual(DEVICE_TOKEN_1.name);
+    expect(result[0].deviceId).toEqual(DEVICE_TOKEN_1.id);
+    expect(result[0].deviceType).toEqual(SESSION_TOKEN_1.uaDeviceType);
+    expect(result[0].sessionTokenId).toEqual('[REDACTED]');
```

Comment:
Maybe it's okay to expose the session token id? But I thought it could also be a security liability. It's probably better if this is never exposed to a support agent?